add-workers/du1-fec1/overlay/default/kustomization.yaml

```diff
@@ -13,10 +13,3 @@ patches:
       kind: BareMetalHost
       name: bmh-day2-worker1
     path: ./patches/bmh-du1-fec1.yaml
-  - target:
-      kind: Provisioning
-      name: provisioning-configuration
-    patch: |-
-      - op: replace
-        path: '/spec/disableVirtualMediaTLS'
-        value: true
```

Comment:
This SMCi system is one of the older generation Cascade Lake based system.  I am not sure if it will be able to provisioned without disabling virtual media TLS.  I know that its successor system (Ice Lake) that we have will not function without `true` for disableVirtualMediaTLS.  The older and newer systems have different BMCs.

Reply:
Since `Provisioning` isn't a namespace scoped, we decided to create it only once. This was solved in #16